CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Unreleased
+
+- Additional tests ([#232](https://github.com/getsentry/sentry-unity/pull/232))
```

Comment:
This doesn't really impact the customer so feel free to use `#skip-changelog` in such cases